examples/basic/version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.56.1"
+      version = ">= 1.61.0"
```

Comment:
basic example should lock into lowest version in the range, which is now `1.61.0`